app/Services/StoreFileService.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brewmap\Services;
+
+use Illuminate\Http\Request;
+
+class StoreFileService
+{
+    protected Object $file;
+
+    public function __construct(Request $request)
```

Comment:
I am inject directly file into function, like this:
```
public function storeFile(string $uploadFolder, UploadedFile $file): string
    {
        $this->path = $file->store($uploadFolder);

        return $this->path;
    }
```

Reply:
I have no idea what comment above means.

Reply:
Sorry,  I will explain more.
I have one method and constructor now in **StoreFileService.php**.
I will put only one method (**storeFile**) to **StoreFIleService** because I inject file to that method from controller method. 
Code below.

UploadAvatarController:
```
public function store(UpdateProfileAvatarRequest $request, StoreFileService $service): JsonResponse
    {
        $this->pathToAvatar = $service->storeFile("users", $request->file("file"));

        return new JsonResponse([
            "path_to_file" => $this->pathToAvatar,
        ]);
    }
```

StoreFileService:
```
public function storeFile(string $uploadFolder, UploadedFile $file): string
    {
        $this->path = $file->store($uploadFolder);

        return $this->path;
    }
```